src/tools/ToolSelectMove.ts

```diff
@@ -123,22 +123,21 @@ export abstract class ToolSelectMove extends BaseMoveTool {
     ];
     const currentSSDots: StuntSheetDot[] =
       GlobalStore.getters.getSelectedStuntSheet.stuntSheetDots;
-    GlobalStore.commit(
-      "setGrapherToolDots",
-      GlobalStore.state.selectedDots.map((index: number) => {
-        const [roundedX, roundedY] = BaseTool.roundCoordinateToGrid([
-          currentSSDots[index].x + deltaX,
-          currentSSDots[index].y + deltaY,
-        ]);
-        return currentSSDots !== null && index < currentSSDots.length
-          ? {
-              x: roundedX,
-              y: roundedY,
-              dotLabelIndex: currentSSDots[index].dotLabelIndex,
-            }
-          : {};
-      })
+    const selectedDotIndicesInSS = GlobalStore.state.selectedDotIndices.filter(
+      (index: number) => currentSSDots && index < currentSSDots.length
     );
+    const grapherToolDots = selectedDotIndicesInSS.map((index: number) => {
+      const [roundedX, roundedY] = BaseTool.roundCoordinateToGrid([
+        currentSSDots[index].x + deltaX,
+        currentSSDots[index].y + deltaY,
+      ]);
+      return new StuntSheetDot({
+        ...currentSSDots[index],
```

Comment:
Huh cool; is this a sort of 'inherit fields from another object' operation?

Reply:
Neat it looks like it just expands the currentSSDots into its fields https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_syntax pretty cool

Reply:
Yup! It's part of ES6 Javascript... Kinda fancy! The `x` and `y` properties below will overwrite whatever came from the object referenced here.